cmd/juju/action/common.go

```diff
@@ -31,17 +31,21 @@ import (
 
 var logger = loggo.GetLogger("juju.cmd.juju.action")
 
-// leaderSnippet is a regular expression for unit ID-like syntax that is used
-// to indicate the current leader for an application.
-const leaderSnippet = "(" + names.ApplicationSnippet + ")/leader"
+const (
+	// leaderSnippet is a regular expression for unit ID-like syntax that is used
+	// to indicate the current leader for an application.
+	leaderSnippet = "(" + names.ApplicationSnippet + ")/leader"
+)
 
-var validLeader = regexp.MustCompile("^" + leaderSnippet + "$")
+var (
+	validLeader = regexp.MustCompile("^" + leaderSnippet + "$")
```

Comment:
I don't see any other use of `leaderSnippet` in the file. Why not just inline it when calling `MustCompile`?

Reply:
No idea, I was just cleaning up lint errors.